src/js/components/FormField/FormField.js

```diff
@@ -245,6 +247,22 @@ const FormField = forwardRef(
       }
     }
 
+    // fileinput handle
+    // use fileinput plain use formfield to drive the border
+    let isFileInputComponent;
+    if (
+      children &&
+      Children.map(children, child => {
+        if (
+          child &&
+          child.type &&
+          FileInputName.indexOf(child.type.displayName) !== -1
```

Comment:
Since we only use this once, it feels like we don't necessary have to define a separate variable for `FileInputName` and could just have it inline here